notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -220,6 +220,56 @@ father and son [Sherman Klump had a heart to heart talk with his father Cletus a
 giant hamster []
 
 
+movie: Hollow Man (2000)
+========================
+
+:: Title
+Hollow Man
+
+:: Description
+Hollow Man is a 2000 science fiction horror film directed by Paul Verhoeven
+and starring Kevin Bacon, Elisabeth Shue and Josh Brolin.
+
+Synopsis: A scientist volunteers to be the first human test subject for a
+serum that renders the user invisible. When his fellow scientists are unable
+to restore him back to normal, he eventually becomes violently insane and goes
+on a murderous rampage.
+
+:: Date
+2000-08-04
+
+:: References
+https://en.wikipedia.org/wiki/Hollow_Man
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+personal invisibility cloak [A main novelty of the film involves the scientist Sebastian Caine injecting himself with an experimental serum that renders its subject invisible and its unfortunate aftermath.]
+invisibility ability [A main novelty of the film surrounds the scientist Sebastian Caine's experience of what it is like to be invisible.]
+
+:: Major Themes
+the human capacity for good and evil [Central to the story was the already narcissistic scientist Sebastian Caine losing all sense of morality once he became invisible and didn't need to worry about facing any consequences for his actions anymore.]
```

Comment:
It sounds to me like there's a "temptation" theme going on here. And/or we could consider introducing a new theme for all those speculative cases when someone finds themself in a position above the law. What might anyone of us not be tempted to do if we were invisible, could stop time, poses someone else's body etc etc?

Reply:
Ah but we do have such a theme!

https://www.themeontology.org/theme.php?name=what%20if%20I%20could%20do%20anything%20I%20wanted%20without%20repercussions

Reply:
Good catch there! I added it as major.

Reply:
So does the good and evil theme really apply? Def:

"The story makes a point about individual human's capability to be both, or alternatingly, good and evil. If explicit reference is not made to this idea, a story is still deemed to exhibit the theme if there is a character who in an exaggerated way is both good and evil."

Sounds to me like he was mostly just wicked.

Reply:
Removed it.